google/cloud/forseti/notifier/notifiers/cscc_notifier.py

```diff
@@ -187,7 +188,8 @@ def find_inactive_findings(new_findings, findings_in_cscc):
             finding_id = finding_list[0]
             to_be_updated_finding = finding_list[1]
 
-            if finding_id not in new_findings_map:
+            if (finding_id not in new_findings_map and
+                    to_be_updated_finding['state'] == 'ACTIVE'):
```

Comment:
Instead of using the `and` operator, can we make this check clearer and simpler by separating it?  Such as below?

```
if to_be_updated_finding['state'] == 'INACTIVE':
    continue

if (finding_id not in new_findings_map:
    <rest of the code>

```

Reply:
updated as suggested